milopy/core.py

```diff
@@ -62,7 +62,12 @@ def make_nhoods(
         knn_graph = adata.obsp[neighbors_key + "_connectivities"].copy()
 
     # Get reduced dim
-    X_dimred = adata.obsm[use_rep]
+    if use_rep == 'X':
+        X_dimred = adata.X
+        if scipy.sparse.issparse(X_dimred)
```

Comment:
Hey @canergen, happy to merge this if you fix syntax error
```suggestion
        if scipy.sparse.issparse(X_dimred):
```